libs/providers/flagd-web/src/lib/flagd-web-provider.spec.ts

```diff
@@ -1,4 +1,4 @@
-import { CallbackClient, Code, ConnectError, PromiseClient } from '@bufbuild/connect-web';
```

Comment:
These imports are deprecated, the inline comments suggested moving to `@bufbuild/connect`. I tested this locally with the playground and it caused no issues.